census/estimator/dataflow_setup.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+'''Dependencies for Dataflow workers.'''
+
+from setuptools import setup, find_packages
+import os
+
+NAME = 'preprocessing'
+VERSION = '1.0'
+REQUIRED_PACKAGES = ['tensorflow-transform==0.9.0']
```

Comment:
nit: CMLE supports now TF 1.12. 
By looking at the TFT matrix, there is a newer version: 0.11.0, does it makes sense to use this one?

Reply:
Done.